exercises/rhde_aw_120/2.1-kickstart-template/README.md

```diff
@@ -67,15 +67,15 @@ ostreesetup --nogpg --url={{ ostree_repo_protocol }}://{{ ostree_repo_host }}:{{
 ```
 {% endraw %}
 
-Here we've convered a few lines to be more dynamic so this template is re-usable. Some of these are fine to store as variables in Controller, while others we'll create a custom credential type and credential for so they're stored securely.
+Here we've converted a few lines to be more dynamic so this template can be re-usable. Some of these changes are to enable the ability to store as variables in Controller, while others, we'll create a custom credential type and credential so that the values can be stored securely.
 
 ### Step 3 - Adding Networking Information
 
-Since we're provisioning these systems over the network, it may be necessary to include networking information in the kickstart to ensure the device is on the network prior to trying to pull the OS.
+Since we're provisioning these systems over the network, it may be necessary to include networking information in the kickstart to ensure the device is on the network prior to trying to pulling the OS.
 
 If wired networking and DHCP is available, then most likely things will just work "out of the box" by specifying `network --bootproto=dhcp --onboot=true` in the kickstart file. If you're using virtualized devices in AWS, this will most likely be your experience as the workshop's VPC will have DHCP available, and the virtual machines will be presented with a "wired" connection.
 
-Wifi connections are not supported in the `network` line of a kickstart, so we'll establish the connection using `nmcli` in the `%pre` section of the kickstart. Additionally, we'll conditionalize this via `if/endif` statements so this section is only present if wireless credentials have been provided.
+Wifi connections are not supported in the `network` line of a kickstart, so we'll establish the connection using `nmcli` in the `%pre` section of the kickstart. Additionally, we'll conditionalize this via jinja `if/endif` statements, so this section is only present if wireless credentials have been provided. Add the following to the beginning of the kickstart file.
 
 {% raw %}
```

Comment:
'{% raw %}' statements are interpreted by Liquid, so having them in the markdown may/may not work